tests/test_cli_compile.py

```diff
@@ -691,8 +691,7 @@ def test_annotate_option(pip_conf, runner, option, expected):
 
 
 @pytest.mark.parametrize(
-    "option, expected",
-    [("--allow-unsafe", "\nsetuptools=="), (None, "\n# setuptools==")],
+    "option, expected", [("--allow-unsafe", "\nsetuptools=="), (None, "\n# setuptools")]
```

Comment:
FTR, would be nice to pin setuptools here as well, but it would take to rewrite the test.

Reply:
I've converted this into an issue: #988.

Reply:
Thanks!